src/openrct2/ride/Vehicle.cpp

```diff
@@ -5506,8 +5521,9 @@ void Vehicle::UpdateSound()
     uint8_t frictionVolume = 255;
     auto frictionId = OpenRCT2::Audio::SoundId::Null;
     // bh screamVolume should be set before hand
-    auto screamId = OpenRCT2::Audio::SoundId::Null;
-    uint8_t screamVolume = 255;
+    SoundIdVolume currentSound = { OpenRCT2::Audio::SoundId::Null, 255 };
```

Comment:
Do you have a preference for the name of this variable? screamSound or currentSound?

Reply:
It's ultimately used for setting sound2. So might as well call it `newSound2`